src/template/volumes/volume1/notebook-test2/nb2.md

```diff
@@ -0,0 +1,53 @@
+---
+jupytext:
```

Comment:
I know this one is still experimental. Maybe we need to think about this part more. In that case would it be better not having this volume part for now? Frankly speaking, I don't know too much about this volume thing at this moment but maybe we can start straight without volume part for now. Then, when we set some kind of standard about how to handle the volume part, then we can add the volume? I am not very sure and I could be wrong, so I would like to ask what you think

Reply:
I can't remember what we settled on for how often new "volumes" (or "issues") will be published. I like the idea of grouping things together like a journal does so it's not a flat list of published notebooks. I can remove this if needed. Just let me know if this preferred.

Reply:
If so, can we remove this volume part from this PR until we decide what to do with it? I guess that something temporary or not decided with what to do might be better to be removed until it gets clearer. Maybe we can create an issue about creating volume as separate issue.